src/build.rb

```diff
@@ -21,6 +21,22 @@
 erb = ERB.new(template_file)
 File.open("../doc/swatches.svg", 'w+') { |file| file.write(erb.result( binding )) }
 
+# REACT JS
+js_lines = []
+js_lines << "var swatches = {"
+color_types.each_value do |color_type|
+	js_lines << "// #{color_type["name"]} (#{color_type["comment"]})"
+	color_type["colors"].each do |key, value|
+			js_lines << "#{key}: 'rgba(#{value.join(',')})',"
+	end
```

Comment:
A more 'standard' way of doing this would be to write the object as JSON. You lose the comments, but since this is basically just writing a file for machine consumption, it's probably fair to leave the 'documentation' in the specimen source

```rb
require "json"

swatches = {}

color_types.each_value do |color_type|
    swatches.merge!(color_type["colors"])  # in-place merge
end

file_contents = "module.exports = #{swatches.to_json};"  # string interpolation instead of concatenation

File.write("../react/colors.js", file_content)  # shorthand for writing a file in Ruby > 1.9.3
```

p.s. my Ruby is really rusty, so the syntax here is assembled from a few StackOverflow posts